t/roundrobin.t

```diff
@@ -44,29 +44,19 @@ __DATA__
 
             for i = 1, 14 do
                 local id = rr:find()
-
-                ngx.say("id: ", id)
```

Comment:
For this sanity test case, maybe we can use three servers with the same weight, and check if they are roundrobined?

Reply:
i plan use three servers with the same weight, check current find different with the last find.  is this ok?

Reply:
@juzipeek
This is OK.